### DISCUSSION TITLE
运行Whisper时404

### DISCUSSION THREAD

**Discussion by sakomasaki** [Ayanaminn/N46Whisper#51] [Q&A]
老师您好，今天进行到Whisper的部分的时候出现了404报错（不知道是不是我这边的问题）

HTTPError: 404 Client Error: Not Found for url: **https://huggingface.co/api/models/guillaumekln/faster-whisper-large-v2/revision/main**

HfHubHTTPError: 404 Client Error: Not Found for url: **https://huggingface.co/api/models/guillaumekln/faster-whisper-large-v2/revision/main** (Request ID: Root=1-645a5e05-2ac57e0d419ec2c1523963c3)

**Comment by Ayanaminn** [comment]
你好，这是昨天晚上开始huggingface不稳定导致的，可以耐心等一下他们维护完毕再用。
